Sources/MapboxMaps/Ornaments/OrnamentsManager.swift

```diff
@@ -177,23 +177,33 @@ public class OrnamentsManager: NSObject {
     }
 
     private func constraints(with view: UIView, position: OrnamentPosition, margins: CGPoint) -> [NSLayoutConstraint] {
+        // handle AutoLayout changes when device character direction changes
+        let currentDeviceLanguage: String! = Locale.current.languageCode
+        let rightToLeft = Locale.LanguageDirection.rightToLeft
+
+        if Locale.characterDirection(forLanguage: currentDeviceLanguage) == rightToLeft {
+            UIView.appearance().semanticContentAttribute = .forceRightToLeft
+            //0 is on the left side
+            _scaleBarView.semanticContentAttribute = .spatial
+        }
+
```

Comment:
I think this can be skipped if `MapboxScaleBarOrnamentView` would use directional anchors(`leading`, `trailing`) and `UITraitCollection.layoutDirection` for checking the direction of layout.
E.g.:
```swift
    private func usesRightToLeftLayout() -> Bool {
        return traitCollection.layoutDirection == .rightToLeft
    }
```